docs/usage/configuration-options.md

```diff
@@ -275,8 +285,11 @@ Solutions:
 
 ## branchName
 
-Warning: it's strongly recommended not to configure this field directly.
-Use at your own risk.
+<!-- prettier-ignore -->
+!!! warning
+    We strongly recommended that you do not configure this field directly.
```

Comment:
Rewritten this sentence so it's our recommendation, not some passive recommendation. 😉